raft/src/main/java/org/apache/kafka/raft/OffsetAndEpoch.java

```diff
@@ -25,6 +25,14 @@ public OffsetAndEpoch(long offset, int epoch) {
         this.epoch = epoch;
     }
 
+    public long offset() {
+        return offset;
+    }
+
+    public int epoch() {
```

Comment:
The fields are public already. I know that the methods are sometimes nicer for lambda expressions. Should we go ahead and make the fields private?

Reply:
Yes. Do you mind if I do this in another PR? I don't want to add too many trivial diffs to this one.

Reply:
@hachikuji https://github.com/apache/kafka/pull/12770